recipes/web.py/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "web.py" %}
+{% set version = "0.40.dev0" %}
```

Comment:
The only alternative I can see is version 0.38, which I tried in https://github.com/conda-forge/staged-recipes/pull/3390/commits/c9307122f2aa723607cc654e8a87f25ba137f38d but as discussed in https://github.com/webpy/webpy/issues/396, that release is not Py3 compatible